Core/Core.csproj

```diff
@@ -14,13 +14,11 @@
 
   <ItemGroup>
     <ProjectReference Include="..\Types\Types.csproj" />
-    <PackageReference Include="AppInsights.WindowsDesktop" Version="2.13.1" />
+    <PackageReference Include="Microsoft.ApplicationInsights" Version="2.14.0" />
```

Comment:
We can't use this package as it doesn't contain the correct telemetry. AppInsights.WindowsDesktop should support other platforms, the key part is that it buffers and persists telemetry.